src/app/projects/add-edit-project/add-edit-project.component.html

```diff
@@ -1,45 +1,38 @@
-<form [formGroup]="myForm" (ngSubmit)="register(myForm)" novalidate>
-
-  <div class="action-container">
+<nav *ngIf="!loading" class="action-container" aria-label="breadcrumb">
+  <div *ngIf="!isEditing">
     <ol class="breadcrumb">
-      <li class="breadcrumb-item" >
+      <li class="breadcrumb-item" aria-current="page">
         <a [routerLink]="['/projects']">All Projects</a>
       </li>
-      <li *ngIf="project" class="breadcrumb-item active" >{{project.name}}</li>
+      <li class="breadcrumb-item active" aria-current="page">
+        Add
+      </li>
     </ol>
-    <div class="btn-container">
-      <button class="btn btn-primary" type="cancel" (click)="onCancel()">Cancel</button>
-      <button  class="btn btn-primary" type="submit"
-          (click)="onSubmit()">Save</button>
-    </div>
   </div>
-  <main class="container-fluid-padding add-edit-form">
-    <div class="spinner-container" *ngIf="loading">
-      <div class="spinner-new rotating"></div>
-    </div>
+  <div *ngIf="isEditing">
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item" aria-current="page">
+        <a [routerLink]="['/projects']">All Projects</a>
+      </li>
+      <li class="breadcrumb-item" aria-current="page">
+        <a [routerLink]="['/p', project._id]">{{project.name}}</a>
+      </li>
+      <li class="breadcrumb-item active" aria-current="page">
+        Edit
+      </li>
+    </ol>
+  </div>
+</nav>
 
-    <div class="action-container" *ngIf="!isEditing">
-      <ol class="breadcrumb">
-        <li class="breadcrumb-item" aria-current="page">
-          <a [routerLink]="['/projects']">All Projects</a>
-        </li>
-        <li class="breadcrumb-item active" aria-current="page">
-          Add
-        </li>
-      </ol>
-    </div>
-    <div *ngIf="isEditing">
-      <!-- <ol class="breadcrumb">
-        <li class="breadcrumb-item" aria-current="page">
-          <a [routerLink]="['/projects']">All Projects</a>
-        </li>
-        <li class="breadcrumb-item" aria-current="page">
-          <a [routerLink]="['/p', projectId]"></a>
-        </li>
-        <li class="breadcrumb-item active" aria-current="page">
-          Edit
-        </li>
-      </ol> -->
+<main class="container-fluid-padding add-edit-form">
+  <div class="spinner-container" *ngIf="loading">
+    <div class="spinner-new rotating"></div>
+  </div>
+  <form [formGroup]="myForm" (ngSubmit)="register(myForm)" novalidate>
```

Comment:
Clicked _create new _project__ and got the page spinner. 

Seeing a console error here:
ERROR Error: formGroup expects a FormGroup instance. Please pass one in.